docs-src/concepts/what-is-cluster-configuration.md

```diff
@@ -0,0 +1,127 @@
+---
+id: what-is-cluster-configuration
+title: What is Cluster configuration?
+sidebar_label: Cluster configuration
+description: Cluster Configuration is the setup and configuration details of your Temporal Cluster, defined using YAML.
+tags:
+  - term
+  - explanation
+---
+
+Cluster configuration is the setup and configuration details of your self-hosted Temporal Cluster, defined using YAML.
+You must define your Cluster configuration when setting up your Temporal Cluster.
+
+Cluster configuration is composed of two types of configurations: Static configuration and Dynamic configuration.
```

Comment:
worth noting that dynamic config file only contains overrides, so the file can be empty and temporal cluster can still work.

Reply:
rearranged the content to highlight this.